docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -367,48 +367,111 @@ Create the required Deployment using:
+instances in the cluster. If the MongoDB intances specify a hostname, then this
+needs to be resolved to the corresponding IP addresses. If the IP address of
+any MongoDB instance changes, we can start a 'rolling upgrade' of NGINX after
+updating the corresponding ConfigMap without affecting availabilty.
 
-Run a container that provides utilities like ``nslookup``, ``curl`` and ``dig``
-on the cluster and query the internal DNS and IP endpoints.
+
+Create the ConfigMap for the whitelist using:
 
 .. code:: bash
+   
+   $ kubectl apply -f nginx-cm.yaml
 
-   $ kubectl run -it toolbox -- image <docker image to run> --restart=Never --rm
+Get the file ``nginx-dep.yaml`` from GitHub using:
 
-It will drop you to the shell prompt.
-Now you can query for the ``mdb`` and ``bdb`` service details.
+.. code:: bash
+   
+   $ wget https://raw.githubusercontent.com/bigchaindb/bigchaindb/master/k8s/nginx/nginx-dep.yaml
+
+Create the NGINX deployment using:
+
+.. code:: bash
+   
+   $ kubectl apply -f nginx-dep.yaml
+
+
+Step 11: Verify the BigchainDB Node Setup
+-----------------------------------------
+
+Step 11.1: Testing Internally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Run a container that provides utilities like ``nslookup``, ``curl`` and ``dig``
+on the cluster and query the internal DNS and IP endpoints.
 
 .. code:: bash
 
-   $ nslookup mdb
-   $ dig +noall +answer _mdb-port._tcp.mdb.default.svc.cluster.local SRV
-   $ curl -X GET http://mdb:27017
-   $ curl -X GET http://bdb:9984
+   $ kubectl run -it toolbox -- image <docker image to run> --restart=Never --rm
 
 There is a generic image based on alpine:3.5 with the required utilities
 hosted at Docker Hub under ``bigchaindb/toolbox``.
 The corresponding Dockerfile is `here
 <https://github.com/bigchaindb/bigchaindb/k8s/toolbox/Dockerfile>`_.
+
 You can use it as below to get started immediately:
 
 .. code:: bash
 
    $ kubectl run -it toolbox --image bigchaindb/toolbox --restart=Never --rm
 
+It will drop you to the shell prompt.
+Now you can query for the ``mdb`` and ``bdb`` service details.
+
+.. code:: bash
+
+   # nslookup mdb-svc
+   # nslookup bdb-svc
+   # nslookup ngx-svc
+   # dig +noall +answer _mdb-port._tcp.mdb-svc.default.svc.cluster.local SRV
+   # dig +noall +answer _bdb-port._tcp.bdb-svc.default.svc.cluster.local SRV
+   # dig +noall +answer _ngx-public-mdb-port._tcp.ngx-svc.default.svc.cluster.local SRV
+   # dig +noall +answer _ngx-public-bdb-port._tcp.ngx-svc.default.svc.cluster.local SRV
+   # curl -X GET http://mdb-svc:27017
+   # curl -X GET http://bdb-svc:9984
+   # curl -X GET http://ngx-svc:80
+   # curl -X GET http://ngx-svc:27017
+
+The ``nslookup`` commands should output the configured IP addresses of the
+services in the cluster
+
+The ``dig`` commands should return the port numbers configured for the
+various services in the cluster.
+
+Finally, the ``curl`` commands test the availability of the services
+themselves.
```

Comment:
When I did the last curl command, i.e. `# curl -X GET http://ngx-svc:27017`, I got the response:
```text
curl: (56) Recv failure: Connection reset by peer
```